### ISSUE TITLE
Pixel Classification "Precondition violation"

### ISSUE THREAD

**Issue by erinmerchant** [ilastik/ilastik#1802]
Hi everyone, 
I have been using ilastik to train images to get a simple segmentation output. It has worked well in the past, but I have changed one thing about the original image and now, when I try and export the segmentation,  I am getting the error message:
Failed to generate export file: (file name)
Precondition violation!
Unable to open file '(file name)'
(C:\conda3\conda-bld\vigra_1513957436976\work\src\impex\tiff.cxx:839)

The only thing I changed where I am now getting the error message is "cropping" the image in ImageJ so there is the brain slice (that I am training), and a black background that has been "cleared" in the ImageJ macro script. Essentially the only thing you see is the brain and its outline as the rest is black. Originally, I was just using the raw data file from the microscope. I am saving the files as .tif, both originally when it worked and when I am getting the error message. I am unsure why I am now getting this error message now, but any help would be appreciated!!

**Comment by erinmerchant** [comment]
Hi Anna,


I originally sent the jpeg because the file was too large as a tiff to send over email. However, I must have been using an existing project and this is what was causing issues because I just tried with a new workflow and it seemed to work fine at least for one image. I am currently using 1.3.0, is there an updated version?

Thanks,

Erin

________________________________

**Comment by boazmohar** [comment]
I don't know if this helps but I ran into the same issue and what happened to me was that I had a partial tif file written (114 pages out of 140) in the output directory when I received this error.
I was able to bypass the problem at least a few times by not having the windows explorer open to the output directory. 
Maybe itastik is opening and appending tif pages and it gets blocked by windows trying to access the same file.
